README.md

```diff
@@ -54,6 +54,7 @@ The following versions of grape, grape-entity and grape-swagger can currently be
 | 0.34.0        | 2.0          | >= 0.16.2 ... < 1.3.0   | >= 0.5.0     | >= 2.4.1      |
 | >= 1.0.0      | 2.0          | >= 1.3.0                | >= 0.5.0     | >= 2.4.1      |
 | >= 2.0.0      | 2.0          | >= 1.7.0                | >= 0.5.0     | >= 2.4.1      |
+| >= 3.0.0      | 2.0          | >= 2.2.0                | >= 0.5.0     | >= 2.4.1      |
```

Comment:
If the change is made backwards-incompatible as suggested, a major release must follow.

Reply:
Let's try not to make a backwards incompatible one and break things only if we have to?